app/main/forms.py

```diff
@@ -3,6 +3,18 @@
 
 from dmutils.forms import StripWhitespaceStringField
 
+from .. import data_api_client
+
+
+class AdminEmailAddressValidator(object):
+
+    def __init__(self, message=None):
+        self.message = message
+
+    def __call__(self, form, field):
+        if not data_api_client.email_is_valid_for_admin_user(field.data):
+            raise validators.StopValidation(self.message)
```

Comment:
Probably something to do at the API level rather than here, but should we check whether the email address belongs to an existing user account? Obviously that user would get an error when they tried to sign up again so not a dealbreaker, but it would save sending an unnecessary email...

Reply:
Hmm interesting, could be done on the apiclient method or api. Probably something to talk to product off. I'll bring it up with tobi

Reply:
Gonna stick this screenshot here for now so I don't lose it

Reply:
![screen shot 2017-11-28 at 09 08 47](https://user-images.githubusercontent.com/3469840/33311226-07044198-d41c-11e7-8990-2b4aa0624aca.png)